tests/e2e/vmservice_late_binding.go

```diff
@@ -269,4 +309,563 @@ var _ bool = ginkgo.Describe("[vmsvc] VM-Service-VM-LateBinding", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		ginkgo.By("Get WFFC stotage class")
+		wffcStorageclass, err := client.StorageV1().StorageClasses().Get(ctx, wffsStoragePolicyName,
+			metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		framework.Logf("wffsStoragePolicyName: %s wffcStorageclass: %s", wffsStoragePolicyName, wffcStorageclass)
+
+		immediateBindingStorageclass, err := client.StorageV1().StorageClasses().Get(ctx, storagePolicyName,
+			metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		framework.Logf("wffsStoragePolicyName: %s wffcStorageclass: %s", wffsStoragePolicyName, wffcStorageclass)
+
+		ginkgo.By("Create a PVC")
+		pvc1, err := createPVC(ctx, client, namespace, nil, "", wffcStorageclass, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Create a PVC")
+		pvc2, err := createPVC(ctx, client, namespace, nil, "", wffcStorageclass, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Create a PVC")
+		pvc3, err := createPVC(ctx, client, namespace, nil, "", immediateBindingStorageclass, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Create a PVC")
+		pvc4, err := createPVC(ctx, client, namespace, nil, "", immediateBindingStorageclass, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Wait for VM images to get listed under namespace and create VM")
+		err = pollWaitForVMImageToSync(ctx, namespace, vmImageName, poll, supervisorClusterOperationsTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		ginkgo.By("Creating VM bootstrap data")
+		secretName := createBootstrapSecretForVmsvcVms(ctx, client, namespace)
+
+		ginkgo.By("Create vm service vm")
+		vm1 := createVmServiceVmWithPvcs(
+			ctx, vmopC, namespace, vmClass, []*v1.PersistentVolumeClaim{pvc1, pvc2, pvc3, pvc4}, vmi,
+			wffsStoragePolicyName, secretName)
+
+		ginkgo.By("Creating loadbalancing service for ssh with the VM")
+		_ = createService4Vm(ctx, vmopC, namespace, vm1.Name)
+
+		ginkgo.By("Wait for VMs to come up and get an IP")
+		vmIp1, err := waitNgetVmsvcVmIp(ctx, vmopC, namespace, vm1.Name)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Waiting PVC to be in bound state")
+		_, err = fpv.WaitForPVClaimBoundPhase(ctx, client, []*v1.PersistentVolumeClaim{pvc1, pvc2, pvc3,
+			pvc4}, pollTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("get VM storage")
+		vmQuotaUsed := getVMStorageData(ctx, vmopC, namespace, vm1.Name)
+		framework.Logf("vmQuotaUsed : %s", vmQuotaUsed)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify PVCs are accessible to the VM")
+		ginkgo.By("Write some IO to the CSI volumes and read it back from them and verify the data integrity")
+		//vm1, err = getVmsvcVM(ctx, vmopC, vm1.Namespace, vm1.Name) // refresh vm info
```

Comment:
Please use this util -> verifyVolumeAccessibilityAndDataIntegrityOnVM

And remove commented code